### PR TITLE
Feature/ds 305 icons header

### PR DESCRIPTION
**Pull request by crishpeen**

### CODE REVIEW

File: packages/web-react/src/components/Header/NavbarClose.tsx [hunk 1]
```diff
@@ -21,7 +20,7 @@ const NavbarClose = (props: unknown): JSX.Element => {
       aria-expanded={isExpanded}
       onClick={handleToggle}
     >
-      <span className={classNames(headerIconClass, headerIconCloseClass)} aria-hidden="true" />
```
Comment (literat) [inline packages/web-react/src/components/Header/NavbarClose.tsx:24]: What about that `aria-hidden="true"`?

**Review by literat — CHANGES_REQUESTED** [review]
LGTM, but not sure about the removing of `aria-hidden="true"`. Maybe you should pass it to the `Icon` component.

**Review by literat — APPROVED** [review]
LGTM

File: packages/web-react/src/components/Header/NavbarClose.tsx [hunk 2]
```diff
@@ -21,7 +20,7 @@ const NavbarClose = (props: unknown): JSX.Element => {
...
+      <Icon name="close" aria-hidden="true" />
```
Comment (adamkudrna) [inline packages/web-react/src/components/Header/NavbarClose.tsx:23]: We talked about moving `aria-hidden="true"` into the `Icon` component, could you please fix it?

File: packages/web/src/scss/components/Header/_theme.scss [hunk 3]
```diff
@@ -38,9 +38,6 @@ $nav-link-current-border-width: 3px;
 $nav-link-current-border-color: tokens.$action-selected-default;
 
 $close-button-size: tokens.$space-1000;
+$close-color: tokens.$action-inverted-default;
```
Comment (adamkudrna) [inline packages/web/src/scss/components/Header/_theme.scss:41]:
```suggestion
$close-button-color: tokens.$action-inverted-default;
```

**Review by adamkudrna — APPROVED** [review]

**Pull request merged**